lib/matplotlib/legend.py

```diff
@@ -130,15 +130,27 @@ def _update_bbox_to_anchor(self, loc_in_canvas):
     corner of the legend in axes coordinates (in which case
     ``bbox_to_anchor`` will be ignored).
 
-bbox_to_anchor : `.BboxBase` or pair of floats
-    Specify any arbitrary location for the legend in `bbox_transform`
-    coordinates (default Axes coordinates).
+bbox_to_anchor : `.BboxBase`, 2-tuple, or 4-tuple of floats
+    Specify location for the legend.  Defaults to `axes.bbox` (if called
+    as a method to `.Axes.legend`) or `figure.bbox` (if `.Figure.legend`).
+
+    Co-ordinates are given by `bbox_transform`, with the default transform
```

Comment:
Was "co-ordinates" hyphenated intentionally?

Reply:
Yes but it need not be if we don’t use that style...

Reply:
Fixed.  I prefer "co-ordinate" to "coordinate", but we are 98% "coordinate" in the docs, so I agree w/ consistency.